register.php

```diff
@@ -0,0 +1,61 @@
+<?php 
+session_start();
+include "user.php";
```

Comment:
utilize `use` statements to include the class, Refer PR for #5

Reply:
added